src/EditorFeatures/CSharpTest/ConvertToRecord/ConvertToRecordTests.cs

```diff
@@ -583,28 +647,34 @@ public class [|C|]
     {
         public int P { get; init; }
         public bool B { get; init; }
+        private const int num = 10;
```

Comment:
At first the behavior here surprised me -- I assumed we were going to delete it since the equality is trivially true since it's a constant. But I guess leaving it is probably best, since the user's code is pretty clearly broken so they probably should look at this. Worth adding a comment to the test to explain the intent here?

Reply:
Yeah, the intent of the test here was that static (including const) comparisons shouldn't be supported, and mainly that it's satisfying the "compare alll instance fields and no more" requirement that the generated implementation creates. I'll add a comment in the test as well.